tests/jballerina-unit-test/src/test/resources/test-src/annotations/annot_access.bal

```diff
@@ -403,6 +403,18 @@ type Student record {|
     string name;
 |};
 
+const Examples EXAMPLES = {
```

Comment:
Can we add bala tests when the 
1. const referred to is from a different module
2. the type used in the const is from a different module

Reply:
Added in [commit](https://github.com/ballerina-platform/ballerina-lang/pull/43163/commits/1453b9364a07f0a63af97c228c2774767e9bdba7).